examples/meta/src/kernel/circular.sg.in

```diff
@@ -7,9 +7,7 @@ Features feats_test = create_features(testdat)
 #![create_features]
 
 #![create_kernel]
-Distance d = create_distance("EuclideanDistance")
```

Comment:
For ex: here, Euclidean distance is set and fed to circular kernel, whereas the formula uses Manhattan distance.